### DISCUSSION TITLE
Make prefix and/or objects public in private bucket

### DISCUSSION THREAD

**Discussion by tunecrew** [minio/minio#15532] [Q&A] [answered]
I'm running Minio locally in a docker container - it serves static files from `my-bucket` to a Python application.

The bucket is private, and the Python apps uses presigned urls, and everything works fine.

I have some files that I need to make publicly accessible without having a presigned url appended, so I created a directory in the root of this bucket called `/public` and moved those files there.

I then called `mc policy set download /data/my-bucket/public` but I still get `AccessDenied` errors when trying to access any files in this directory.

What am I missing here?

**Comment by harshavardhana** [comment, marked as answer]
You need to first configure `mc` to talk to MinIO and then use the subcommand to set anonymous bucket access

**Reply by tunecrew** [reply]
Worked - thanks.

A bit unclear in the docs how to go about this - I had better luck here https://www.scaleway.com/en/docs/storage/object/api-cli/installing-minio-client/
